modules/centrality-algorithms/index.adoc

```diff
@@ -0,0 +1,22 @@
+= Centrality Algorithms
+:description: Overview of centrality algorithms.
+
+Centrality algorithms are one way to determine the importance of a vertex to a graph, based on the importance of a vertex to other vertices.
+These algorithms assign a score to a vertex or edge based on some concept of being close to the center of connection-based activity.
```

Comment:
A centrality algorithm usually assigns scores to vertices or edges based on how close they are to the center of connection-based activity. 
The definition of closeness and what constitutes the center of connection-based activity depend on the algorithm.

I see you used the definition in the GDS coding guidelines document. However, that document is internal, so it tends to use a more informal tone, and can be less specific because there's more context. When we publish a publish document, let's make sure our sentences sound more "sure" and avoid phrasing such as "some concept".